src/pages/UserAccount/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -0,0 +1,340 @@
+            className="pf-c-title pf-m-md"
+          >
+            Login
+          </h4>
+          <input
+            aria-invalid={false}
+            aria-label="readonly profile login"
+            className="pf-c-form-control"
+            disabled={true}
+            onBlur={[Function]}
+            onChange={[Function]}
+            onFocus={[Function]}
+            readOnly={false}
+            required={false}
+            type="text"
+            value=""
+          />
+        </div>
+      </div>
+      <div
+        className="pf-l-flex pf-m-column"
+      >
+        <div
+          className=""
+        >
+          <h4
+            className="pf-c-title pf-m-md"
+          >
+            Email
+          </h4>
+          <input
+            aria-invalid={false}
+            aria-label="readonly profile email"
+            className="pf-c-form-control"
+            disabled={true}
+            onBlur={[Function]}
+            onChange={[Function]}
+            onFocus={[Function]}
+            readOnly={false}
+            required={false}
+            type="text"
+          />
+        </div>
+      </div>
+      <div
+        className="pf-l-flex pf-m-column"
+      >
+        <div
+          className=""
+        >
+          <h4
+            className="pf-c-title pf-m-md"
+          >
+            First Name
+          </h4>
+          <input
+            aria-invalid={false}
+            aria-label="readonly profile First Name"
+            className="pf-c-form-control"
+            disabled={true}
+            onBlur={[Function]}
+            onChange={[Function]}
+            onFocus={[Function]}
+            readOnly={false}
+            required={false}
+            type="text"
+            value=""
+          />
+        </div>
+      </div>
+      <div
+        className="pf-l-flex pf-m-spacer-2xl pf-m-column"
+      >
+        <div
+          className=""
+        >
+          <h4
+            className="pf-c-title pf-m-md"
+          >
+            Last Name
+          </h4>
+          <input
+            aria-invalid={false}
+            aria-label="readonly profile Last Name"
+            className="pf-c-form-control"
+            disabled={true}
+            onBlur={[Function]}
+            onChange={[Function]}
+            onFocus={[Function]}
+            readOnly={false}
+            required={false}
+            type="text"
+            value=""
+          />
+        </div>
+      </div>
+      <div
+        className="pf-l-flex"
+      >
+        <button
+          aria-disabled={true}
+          aria-label="edit-account-info"
+          className="pf-c-button pf-m-primary pf-m-disabled"
+          data-ouia-component-id="OUIA-Generated-Button-primary-2"
+          data-ouia-component-type="PF4/Button"
+          data-ouia-safe={true}
+          disabled={true}
+          onClick={[Function]}
+          role={null}
+          tabIndex={null}
+          type="button"
+        >
+          Edit
```

Comment:
Is it sync with the current state where it can be fully disabled, like in case of Openshift.io, where users don't have access to user management system (like keycloak) ?

Reply:
Tested. It is!
On che.openshift.io edit is disabled, on local Che edit leads to keycloak